### ISSUE TITLE
dosbox-x -debug - is this expected behavior?

### ISSUE THREAD

**Issue by nelgin** [joncampbell123/dosbox-x#5147]
### Question

Hi all,

I noticed some unusual behavior when scrolling up in the code from a breakpoint. I seem to lose lines occasionally. Example:

The code broke at
202C:000008EA E2FA                loop 000008E6 ($-6)

Then I cursor up
202C:000008E8 18FF                sbb  bh,bh

I cursor up again
The 8E8 line is overwritten by
202C:000008E7 E818FF              call 00000802 ($-e8)

This keeps happening when I scroll up. Is this expected behavior? 

### Have you checked that no similar question(s) exist?

- [X] I have searched and didn't find any similar question.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by joncampbell123** [comment]
It is. This is the same behavior as DOSBox SVN and other forks. Scrolling up is accomplished by decoding instructions until the longest one is found.

It's obviously not the best way to do it, but it kinda works.

**Comment by nelgin** [comment]
Definitely not the best. Kind of makes it hard to follow the follow. I guess I'll deal with it or I can cross reference with a decompile I have. Thanks.
